### PR TITLE
Adds the value property to the checkboxes and a README to the switch webcomponent

### PR DESCRIPTION
**Pull request by rowa-audil**
### <strong>Pull Request</strong>

The native checkbox support a attribute value.
This PR adds that to both the fluid-checkbox and -switch.
Also adds the missing README to the Switch

#### Type of PR

Feature (non-breaking change which adds functionality)
Documentation update (changes to documentation)

#### Checklist

- [x] I have read the CONTRIBUTING doc and I follow the PR guidelines
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: libs/fluid-elements/checkbox/src/lib/checkbox.spec.ts [hunk 1]
```diff
@@ -218,6 +218,27 @@ describe('Fluid checkbox', () => {
...
+    });
+
+    it('should set the value to customvalue when the property is set', async () => {
+      fixture.value = 'customvalue';
+      await tick();
+      const shadowedInput = fixture.shadowRoot?.querySelector('input');
+      expect(shadowedInput?.getAttribute('value')).toBe('customvalue');
+    });
+
+    it('should set the value to customvalue when the attribute is set', async () => {
```
Comment (lukasholzer) [inline libs/fluid-elements/checkbox/src/lib/checkbox.spec.ts:234]: Please add a test case that is setting the value via clicking? Then you cover all the scenarios for how to set the value.

File: libs/fluid-elements/switch/src/lib/switch.spec.ts [hunk 2]
```diff
@@ -150,6 +150,27 @@ describe('Fluid switch', () => {
...
+      expect(shadowedInput?.getAttribute('value')).toBe('on');
+    });
+
+    it('should set the value to customvalue when the property is set', async () => {
+      fixture.value = 'customvalue';
+      await tick();
+      const shadowedInput = fixture.shadowRoot?.querySelector('input');
+      expect(shadowedInput?.getAttribute('value')).toBe('customvalue');
+    });
+
```
Comment (lukasholzer) [inline libs/fluid-elements/switch/src/lib/switch.spec.ts:165]: please add a test with clicking here as well.

File: libs/fluid-elements/switch/src/lib/switch.spec.ts [hunk 3]
```diff
@@ -150,6 +150,27 @@ describe('Fluid switch', () => {
     });
   });
 
+  describe('value', () => {
+    it('should set the default value to on', async () => {
```
Comment (lukasholzer) [inline libs/fluid-elements/switch/src/lib/switch.spec.ts:154]: please add a test scenario where no value is present

File: libs/fluid-elements/checkbox/src/lib/checkbox.spec.ts [hunk 4]
```diff
@@ -218,6 +218,27 @@ describe('Fluid checkbox', () => {
     });
   });
 
+  describe('value', () => {
+    it('should set the default value to on', async () => {
```
Comment (lukasholzer) [inline libs/fluid-elements/checkbox/src/lib/checkbox.spec.ts:222]: please add a test scenario where no value is present

File: apps/fluid-dev/src/pages/checkbox/checkbox-page.component.html [hunk 5]
```diff
@@ -3,6 +3,7 @@
   [disabled]="disabled"
   [checked]="_isChecked()"
   [indeterminate]="_isIndeterminate()"
+  [value]="'checkbox'"
```
Comment (tomheller) [inline apps/fluid-dev/src/pages/checkbox/checkbox-page.component.html:6]: No need to bind that property, if you don't bind it to a variable.
```suggestion
  value="checkbox"
```

File: apps/fluid-dev/src/pages/switch/switch-page.component.html [hunk 6]
```diff
@@ -1,6 +1,7 @@
 <fluid-switch
   [disabled]="disabled"
   [checked]="checked"
+  [value]="'switch'"
```
Comment (tomheller) [inline apps/fluid-dev/src/pages/switch/switch-page.component.html:4]: No need to bind that property, if you don't bind it to a variable.
```suggestion
  value="switch"
```

File: libs/fluid-elements/checkbox/src/lib/checkbox.spec.ts (hunk 1 above)
Comment (tomheller) [reply libs/fluid-elements/checkbox/src/lib/checkbox.spec.ts:234]: Value of a checkbox cannot be set vie clicking. The value does not represent the state of the checkbox (as in checked / unchecked). Example from mdn

> A DOMString representing the value of the checkbox. This is never seen on the client-side, but on the server this is the value given to the data submitted with the checkbox's name. Take the following example:

```html
<form>
  <div>
    <input type="checkbox" id="subscribeNews" name="subscribe" value="newsletter">
    <label for="subscribeNews">Subscribe to newsletter?</label>
  </div>
  <div>
    <button type="submit">Subscribe</button>
  </div>
</form>
```

File: libs/fluid-elements/checkbox/src/lib/checkbox.ts [hunk 7]
```diff
@@ -86,6 +86,10 @@ export class FluidCheckbox extends LitElement {
   })
   disabled = false;
 
+  /** The value attribute of the native input element */
+  @property({ type: String, reflect: false })
```
Comment (tomheller) [inline libs/fluid-elements/checkbox/src/lib/checkbox.ts:90]: The value can be reflected, it is also when done natively.
```suggestion
  @property({ type: String, reflect: true })
```

File: libs/fluid-elements/checkbox/src/lib/checkbox.ts [hunk 8]
```diff
@@ -86,6 +86,10 @@ export class FluidCheckbox extends LitElement {
...
+  value = 'on'; // Default string for the value is 'on' from MDN (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox#Additional_attributes)
```
Comment (tomheller) [inline libs/fluid-elements/checkbox/src/lib/checkbox.ts:91]: Nitpick:

```suggestion
    // Default string for the value is 'on' from MDN (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox#Additional_attributes)
    value = 'on';
```

File: libs/fluid-elements/checkbox/src/lib/checkbox.spec.ts (hunk 4 above)
Comment (tomheller) [reply libs/fluid-elements/checkbox/src/lib/checkbox.spec.ts:222]: This is the test where no value is present. The default based on the MdnDocumentation is `on`. 
@rowa-audil maybe reword the tests `it` to `should have the correct default set`.

File: libs/fluid-elements/switch/src/lib/switch.spec.ts (hunk 3 above)
Comment (tomheller) [reply libs/fluid-elements/switch/src/lib/switch.spec.ts:154]: This is the test where no value is present. The default based on the MdnDocumentation is on.
@rowa-audil maybe reword the tests it to should have the correct default set.

File: libs/fluid-elements/switch/src/lib/switch.spec.ts (hunk 2 above)
Comment (tomheller) [reply libs/fluid-elements/switch/src/lib/switch.spec.ts:165]: Same here as on the checkbox, there is no way to set the value by clicking.

File: libs/fluid-elements/switch/src/lib/switch.ts [hunk 9]
```diff
@@ -46,7 +62,11 @@ export class FluidSwitch extends LitElement {
   @property({ type: Boolean, reflect: true })
   disabled = false;
 
-  /** Whether the switch is active or not */
+  /** The value attribute of the native input element */
+  @property({ type: String, reflect: false })
```
Comment (tomheller) [inline libs/fluid-elements/switch/src/lib/switch.ts:66]: Value can be reflected.
```suggestion
  @property({ type: String, reflect: true })
```
